packages/cli/src/lib/system/DockerCompose.ts

```diff
@@ -1,12 +1,13 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { Logger } from "../logging";
 import { getDockerFileLock } from "./docker";
 
 import path from "path";
 import Commands, { IDockerComposeOptions } from "docker-compose";
 import { InfraManifest } from "@polywrap/polywrap-manifest-types-js";
 
 export class DockerCompose {
-  private _dockerLock = getDockerFileLock();
+  private _dockerLock = getDockerFileLock(new Logger({}));
```

Comment:
Any particular reason for passing an empty Logger here instead of an actual one?